docs/user/get-started.md

```diff
@@ -0,0 +1,333 @@
+# Get Started
+
+For the one who is interested in Vald, this article will show you a way to deploy and run Vald on k8s.
+This article uses Scylla DB as the backend data store for metadata-management and backup-manager.
+Fashion-mnist is used as an example of a dataset.
+
+1. [About](#About)
+    1. [Main Features](#Main-Features)
+    2. [Requirements](#Requirements)
+2. [Starting Vald on k8s cluster](#Starting-Vald-on-k8s-cluster)
+    1. [Deploy](#Deploy)
+    2. [Run](#Run)
+3. [Advanced](#Advanced)
+
+## About
+
+Vald is a distributed highly scalable and fast approximate nearest neighbor dense vector search engine.<br>
+It provides the search result of any input (e.g. word, sentence, image and etc.) which is coverted to a multi-dimensional vector when searching the neighbor.<br>
```

Comment:
maybe there's better expression :thinking:

Reply:
@rinx 
How do you think about the below suggestion? 

```suggestion
Vald enables large scale image similarity search and large scale specific object (e.g. text, video, face and etc.) similarity search.<br>
```

Reply:
hmm.
How about the followings?

```
Using Vald, you can construct your own similarity search engine (e.g. image, text, video, face, etc...) just by extracting feature vector.
```

Reply:
Hmm, both of them sounds good to me
In rinx's opinion, I'd like to add the word "large scale" it's vald's core feature

Reply:
Well. I just think a sentence. How about the following?

`Using Vald, you can construct your similarity search engine (e.g. image, text, video, face, etc...) just by extracting feature vector, even if you handle large scale specific object.`